src/flash/nor/km1m7xx.c

```diff
@@ -0,0 +1,588 @@
+/***************************************************************************
+ * (C) Nuvoton Technology Corporation Japan 2021                           *
+ *   yamaguchi.yoshikazu@nuvoton.com                                       *
```

Comment:
Please add your full name prior to the mail address.

Reply:
Corrected to "Full name <email address>".